tutorials/tour/_posts/2017-02-13-pattern-matching.md

```diff
@@ -10,38 +10,129 @@ num: 12
 
 next-page: singleton-objects
 previous-page: case-classes
+prerequisite-knowledge: case-classes, string-interpolation, subtyping
 ---
 
-Scala has a built-in general pattern matching mechanism. It allows to match on any sort of data with a first-match policy. 
-Here is a small example which shows how to match against an integer value:
+Pattern matching is a mechanism for checking a value for a pattern. It is a more powerful version of the `switch` statement in Java and it can likewise be used in place of a series of if/else statements.
```

Comment:
suggested for the first sentence:

Pattern matching is a mechanism for checking a value against a pattern. A successful match can also deconstruct a value into its constituent parts.